### ISSUE TITLE
Element-Desktop does not start after update to 1.11.4

### ISSUE THREAD

**Issue by FizzyTea** [vector-im/element-web#23178]
### Steps to reproduce

1. On Xubuntu 20.04 updated element-desktop to 1.11.4 using apt.
2. Opened app from GUI, no response, app does not open.
3. Run command `element-desktop` from terminal. Output below

```
Couldn't find webapp files in any of: 
	/opt/Element/resources/app.asar/webapp.asar
	/opt/Element/resources/webapp.asar
	/opt/Element/resources/app.asar/webapp
	/opt/Element/resources/webapp
App setup failed: exiting Error: Failed to find webapp files
    at /opt/Element/resources/app.asar/lib/electron-main.js:145:15
    at Generator.throw (<anonymous>)
    at rejected (/opt/Element/resources/app.asar/lib/electron-main.js:43:65)
```

#### What did you expect?
The app to open and run as normal

#### What happened instead?
Nothing

### Operating system

Xubuntu 20.02

### Application version

Element version 1.11.4

### How did you install the app?

apt package manager

### Will you send logs?

No

**Comment by FizzyTea** [comment]
I can see this has been addressed with an update.

**Comment by t3chguy** [comment]
For anyone else hitting this issue, it is fixed in 1.11.4-1 - it happened due to a packaging failure
